fireapp/lib/firebase/firebase_options.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart'
+    show defaultTargetPlatform, kIsWeb, TargetPlatform;
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
```

Comment:
I am assuming you copied this from somewhere - that is fine - but the project already has no target for anything but mobile so please just remove the web and non-mobile exceptions (just leave the default case).